exe/cli/cmd/ffs_id.go

```diff
@@ -0,0 +1,37 @@
+package cmd
+
+import (
+	"context"
+	"time"
+
+	"github.com/caarlos0/spin"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func init() {
+	ffsIDCmd.Flags().StringP("token", "t", "", "FFS auth token")
+
+	ffsCmd.AddCommand(ffsIDCmd)
+}
+
+var ffsIDCmd = &cobra.Command{
+	Use:   "id",
+	Short: "Retuns the FFS instance id",
+	Long:  `Retuns the FFS instance id`,
+	PreRun: func(cmd *cobra.Command, args []string) {
+		err := viper.BindPFlags(cmd.Flags())
+		checkErr(err)
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		ctx, cancel := context.WithTimeout(context.Background(), time.Second*60)
+		defer cancel()
+
+		s := spin.New("%s Getting FFS instance id...")
```

Comment:
I wonder if we should change `instance` with `API`.
I doubt sometimes about the `API` name in general... something to think about.

Reply:
Hmm yea, I'm not sure API is ideal either. I think they're both not so clear. I'll leave it as is for now and let's think about the best names.